docs/src/plotting.md

```diff
@@ -138,27 +138,12 @@ Lighthouse.plot_binary_discrimination_calibration_curves(
 )
 ```
 
-# PRG curves
-
-```@docs
-Lighthouse.plot_prg_curves
-```
-
-```@example 1
-using Lighthouse: plot_prg_curves
-plot_prg_curves(
-    curves,
-    stable_rand(5),
-    classes
-)
-```
-
 ## Theming
 
 ```@example 1
 # The plots with only a series don't have a special keyword
 with_theme(SeriesPlot = series_theme) do
-    plot_prg_curves(
+    plot_pr_curves(
```

Comment:
@SimonDanisch --- I'm not confident in this change, def take a look here.

Reply:
I think that just needs to go ;)